### PR TITLE
Add task 375 from paper "Exploring the Role of Argument Structure in Online Debate Persuasion"

### PR DESCRIPTION
**Pull request by Palipoor**

### CODE REVIEW

File: tasks/task375_classify_type_of_sentence_in_debate.json [hunk 1]
```diff
@@ -0,0 +1,2541 @@
...
+            "input": "topic: 4a. Campaign Finance Reform sentence: As it turns out, the laws we have work and people don't forgive those in the pockets of corporations.",
+            "output": "Fact",
+            "explanation": "This sentence is an objective proposition stating a fact."
+        },
+        {
+            "input": "topic: Abortion Is Bad sentence: It also bad because God made all of us so only he can end our lives.",
+            "output": "Value",
+            "explanation": "The sentence is expressing a value judgement without suggesting a policy."
+        }
+    ],
```
Comment (danyaljj) [inline tasks/task375_classify_type_of_sentence_in_debate.json:39]: Wondering if you can add more examples here? (like at least one example from each category).

File: tasks/task375_classify_type_of_sentence_in_debate.json [hunk 2]
```diff
@@ -0,0 +1,2541 @@
...
... a specific course of action to be taken. It typically contains modal verbs like \"should\" and \"ought to\". It cannot be directly proved with objective evidence, and a proper type of support is a logical reason from which the proposition can be inferred.\nValue: This refers to a sentence containing value judgments without making specific claims about what should be done (If so, then it is a Policy sentence.). Because of the subjectivity of value judgments, it cannot be proved directly with objective evidence.\nFact: This refers to an objective proposition expressing or dealing with facts or conditions as perceived without distortion by personal feelings, prejudices, or interpretations. A Fact sentence has a truth value that can be verified with objective evidence that may be available at the time the claim is made; predictions about future are considered unverifiable.\nTestimony: This refers to an objective sentence about the author's personal state or experience. Evidence for Testimony is not publicly available in most cases.\nReference: This refers to a reference to a source of objective evidence. It is typically a citation or a URL of a published work from a renowned source.",
```
Comment (danyaljj) [inline tasks/task375_classify_type_of_sentence_in_debate.json:12]: The definition looks good to me.

File: tasks/task375_classify_type_of_sentence_in_debate.json (hunk 1 above)
Comment (Palipoor) [reply tasks/task375_classify_type_of_sentence_in_debate.json:39]: sure! ok.

**Comment by danyaljj** [comment]
Thanks! Looks good to me!

**Pull request merged**